### PR TITLE
kubevirt: VMWizard - implement footer and the wizard flow 

### PR DESCRIPTION
**Pull request by suomiy**
depends on:
- [x] #2452  

implement logic (disabling, jumps) for buttons and nav list

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard-footer.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+type WizardContext = {
+  onNext: () => void;
+  onBack: () => void;
+  onClose: () => void;
+  activeStep: WizardStep;
+  goToStepById: (id: number | string) => void;
+};
+type CreateVMWizardFooterComponentProps = {
+  stepData: any;
+  createVmText: string;
```
Comment (rawagner) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard-footer.tsx]:
```suggestion
  createVMText: string;
```

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard-footer.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+};
+
+const CreateVMWizardFooterComponent: React.FC<CreateVMWizardFooterComponentProps> = ({
+  stepData,
+  createVmText,
```
Comment (rawagner) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard-footer.tsx]:
```suggestion
  createVMText,
```

**Comment by suomiy** [comment]
fixed

**Pull request merged**